### PR TITLE
Make all properties in the model as optional

### PR DESCRIPTION
**Pull request by sarun-altmtrk**

### CODE REVIEW

File: Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift [hunk 1]
```diff
@@ -32,6 +32,11 @@ public final class HyperwalletPrepaidCard: HyperwalletTransferMethod {
         return getField(TransferMethodField.cardType.rawValue)
     }
 
+    /// The prepaid card's cvv
+    public var cvv: String? {
```
Comment (syyang-hw) [inline Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift]: Where did you see this property for prepaid card?

File: Sources/HyperwalletError.swift [hunk 2]
```diff
@@ -261,15 +261,15 @@ internal struct ErrorTypeHelper {
     static func graphQlErrors(errors: [GraphQlError]) -> HyperwalletErrorType {
         var hyperwalletErrors = [HyperwalletError]()
         for graphQlError in errors {
-            let hyperwalletError = HyperwalletError(message: graphQlError.message,
+            let hyperwalletError = HyperwalletError(message: graphQlError.message ?? "Unexpected Error",
```
Comment (syyang-hw) [inline Sources/HyperwalletError.swift]: Please use localized message when graphQlError.message is nil

File: Sources/HyperwalletError.swift [hunk 3]
```diff
@@ -261,15 +261,15 @@ internal struct ErrorTypeHelper {
...
                                                     code: graphQlError.extensions?.code ?? "UNEXPECTED_ERROR",
                                                     fieldName: nil,
                                                     relatedResources: nil)
             hyperwalletErrors.append(hyperwalletError)
             os_log("GraphQl response contains error: %s",
                    log: OSLog.graphQl,
                    type: .info,
-                   graphQlError.message)
+                   graphQlError.message ?? "Unexpected Error")
```
Comment (syyang-hw) [inline Sources/HyperwalletError.swift]: localized string

File: Tests/TransferMethod/HyperwalletPrepaidCardTests.swift [hunk 4]
```diff
@@ -47,21 +47,24 @@ class HyperwalletPrepaidCardTests: XCTestCase {
...
+        firstPrepaidCard?.setField(key: HyperwalletTransferMethod.TransferMethodField.cvv.rawValue, value: "234")
         XCTAssertEqual(firstPrepaidCard?.type, "PREPAID_CARD", "The type should be PREPAID_CARD")
         XCTAssertEqual(firstPrepaidCard?.token, "trm-123", "The token should be trm-123")
         XCTAssertEqual(firstPrepaidCard?.status, "DEACTIVATED", "The status should be DEACTIVATED")
         XCTAssertEqual(firstPrepaidCard?.transferMethodCountry, "CA", "The country code should be CA")
         XCTAssertEqual(firstPrepaidCard?.transferMethodCurrency, "USD", "The currency code should be USD")
 
         XCTAssertEqual(firstPrepaidCard?.cardNumber, "************6198", "The cardNumber should be ************6198")
+        XCTAssertEqual(firstPrepaidCard?.cardType, "VIRTUAL")
+        XCTAssertEqual(firstPrepaidCard?.cvv, "234")
```
Comment (syyang-hw) [inline Tests/TransferMethod/HyperwalletPrepaidCardTests.swift]: prepaid cards do not have cvv

File: Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift (hunk 1 above)
Comment (jsaini-hw) [reply Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift]: We shouldn't expose cvv. API should never return cvv in the response so kindly remove it

File: Tests/TransferMethod/HyperwalletPrepaidCardTests.swift [hunk 5]
```diff
@@ -47,21 +47,24 @@ class HyperwalletPrepaidCardTests: XCTestCase {
 
         XCTAssertNotNil(prepaidCardList?.links, "The `links` should be not nil")
 
-        let firstPrepaidCard = prepaidCardList?.data.first
+        let firstPrepaidCard = prepaidCardList?.data?.first
+        firstPrepaidCard?.setField(key: HyperwalletTransferMethod.TransferMethodField.cvv.rawValue, value: "234")
```
Comment (jsaini-hw) [inline Tests/TransferMethod/HyperwalletPrepaidCardTests.swift]: Remove this line

File: Tests/TransferMethod/HyperwalletPrepaidCardTests.swift (hunk 4 above)
Comment (jsaini-hw) [reply Tests/TransferMethod/HyperwalletPrepaidCardTests.swift]: remove this line

**Pull request merged**